resources/building-mobile-extensions/event-processing.md

```diff
@@ -8,7 +8,9 @@ An event is composed of the following information:
 * Event data, which is the structured data that is associated with this particular event.   This data is retrievable as a JSON blob.
 * Versioning information, which is used to synchronize a group of events with the associated shared data, is not directly exposed to extensions.  This information is implicitly used when getting shared states.
 
-**Tip**: Extensions use the `ACPExtensionEvent` \(iOS\) / `Event` \(Android\) class to create events and access the information the modules hold.
+**Tip**: Extensions use the ACPExtensionEvent (iOS) / Event (Android) class to create events to dispatch to the extensions in order to make requests or retrieve data.
+
+In the Adobe Experience Cloud Platform SDK, the strings  like extension name, event type and source strings are converted and stored in lowercase. When you compare strings use java.lang.String.equalsIgnoreCase() method.
```

Comment:
I would use something similar with the above comment:
`The event type and event source may be lower-cased by the Mobile SDK as it compares them as case insensitive values. It is recommended to use ignore case string comparison when checking the event type and event source of the event.`

Reply:
Please use the reworded text I added above:

"The event type and event source that are associated with the received event can be in lower case because the Mobile SDK compares the type and the source as case insensitive values. When you check the event type and event source of the event, we recommend that you use the ignore case string comparison."

Reply:
Thank you, @rekrishn 💯